guides/common/modules/proc_integrating-idm-dns-with-tsig-authentication.adoc

```diff
@@ -0,0 +1,64 @@
+[id="integrating-idm-dns-with-tsig-authentication"]
+= Integrating IdM DNS with TSIG authentication
+
+You can configure an IdM server to use the secret key transaction authentication for DNS (TSIG) technology that uses the `rndc.key` key file for authentication.
+The TSIG protocol is defined in https://tools.ietf.org/html/rfc2845[RFC2845].
+
+
+.Prerequisites
+
+* You must ensure the IdM server is deployed and the host-based firewall is configured correctly.
+ifdef::satellite[]
```

Comment:
I wouldn't mind this being generic since it's just linking to product documentation, but I know orcharhino generally doesn't do that.
```suggestion
ifndef::orcharhino[]
```